### PR TITLE
wired up video urls

### PR DESCRIPTION
**Pull request by hcmgr**

### CODE REVIEW

File: src/main/java/com/deco/tickexperience/model/entity/Sport.java [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package com.deco.tickexperience.model.entity;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.OneToMany;
+import jakarta.persistence.*;
```
Comment (Wo-m) [inline src/main/java/com/deco/tickexperience/model/entity/Sport.java:4]: I imagine your ide did this, by default i think if you import >=5 from a package it does a insert all. Can change it somewhere

File: src/main/resources/db/changelog/mock_events.sql [hunk 2]
```diff
@@ -12,4 +12,4 @@ INSERT INTO Event (SportId, VenueId, Name, StartTime, EndTime, ImageURL) VALUES
 -- Rugby
 INSERT INTO Event (SportId, VenueId, Name, StartTime, EndTime, ImageURL, Gender) VALUES (3, 2, 'Pool 1 Rounds', '2032-08-05T11:00:00', '2032-08-05T14:00:00', 'https://cdn.pixabay.com/photo/2016/06/15/01/11/soccer-1457988_1280.jpg', 'M');
 INSERT INTO Event (SportId, VenueId, Name, StartTime, EndTime, ImageURL, Gender) VALUES (3, 2, 'Pool 2 Rounds', '2032-08-06T11:00:00', '2032-08-07T14:00:00', 'https://images.pexels.com/photos/3148452/pexels-photo-3148452.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1', 'M');
-INSERT INTO Event (SportId, VenueId, Name, StartTime, EndTime, ImageURL, Gender) VALUES (3, 3, 'Finals Rounds', '2032-08-07T11:00:00', '2032-08-07T14:00:00', 'https://images.unsplash.com/photo-1650676182612-a2a5a435ce39?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1yZWxhdGVkfDE0fHx8ZW58MHx8fHx8&w=1000&q=80', 'M');
\ No newline at end of file
+INSERT INTO Event (SportId, VenueId, Name, StartTime, EndTime, ImageURL, Gender) VALUES (3, 2, 'Finals Rounds', '2032-08-07T11:00:00', '2032-08-07T14:00:00', 'https://images.unsplash.com/photo-1650676182612-a2a5a435ce39?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1yZWxhdGVkfDE0fHx8ZW58MHx8fHx8&w=1000&q=80', 'M');
```
Comment (Wo-m) [inline src/main/resources/db/changelog/mock_events.sql:15]: This was on purpose, have the finals at a different venue. I believe 3 is the gabba and 2 is suncorp so both capable of hosting a football match.

**Review by Wo-m — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/mock_events.sql (hunk 2 above)
Comment (hcmgr) [reply src/main/resources/db/changelog/mock_events.sql:15]: oh soooooooo

File: src/main/java/com/deco/tickexperience/model/entity/Sport.java (hunk 1 above)
Comment (hcmgr) [reply src/main/java/com/deco/tickexperience/model/entity/Sport.java:4]: REMOVING IT FUCKS IT, 'LLOW IT LAD

**Review by Wo-m — APPROVED** [review]

**Pull request merged**